### DISCUSSION TITLE
Coding sequence as additional output file

### DISCUSSION THREAD

**Discussion by fgaascht** [Gaius-Augustus/Augustus#338] [Q&A] [answered]
Hi,

I have been using Augustus in web interface for gene prediction in Fungi.
As I recently started learning programming and more specifically Python, I decided to give a try to a local version of Augustus that I installed on my Ubuntu machine.

My results are exactly what I am expecting but I was wondering if there is a way or command to also generate text or fasta files with the results like I can get directly from Augustus web interface (predicted coding sequence, predicted amino sequence).

Thank you in advance.

**Comment by MarioStanke** [comment, marked as answer]
The sequences are by default included as a comment to the GFF and can be extracted from the output of `augustus` with `getAnnoFasta.pl`.

**Reply by fgaascht** [reply]
Hi Mario,

Thank you for the information.
